test/testUtils/TestSupport.scala

```diff
@@ -71,6 +72,16 @@ trait TestSupport extends UnitSpec with GuiceOneAppPerSuite with BeforeAndAfterA
   implicit val mockImplicitDateFormatter: ImplicitDateFormatterImpl = new ImplicitDateFormatterImpl(languageUtils)
 
   implicit val messages: Messages = app.injector.instanceOf[MessagesApi].preferred(FakeRequest())
+  val messages_expected: Properties = loadPropertiesFile("/messages_expected")
+
+  private def loadPropertiesFile(classpathPropertiesFile: String): Properties = {
```

Comment:
you should be able to defined a custom Lang resource and user existing underlying logic for this by switching from default language to custom; no need for this method